src/store/selectors/valImagesSelector.ts

```diff
@@ -1,12 +1,13 @@
 import { Project } from "../../types/Project";
 import { Image } from "../../types/Image";
+import { Partition } from "../../types/Partition";
 
 export const valImagesSelector = ({
   project,
 }: {
   project: Project;
 }): Array<Image> => {
   return project.images.filter((image: Image) => {
-    return image.partition === 1;
+    return image.partition === Partition.Training;
```

Comment:
Shouldn't this check for Partition.Validation, instead of Partition.Training?

Reply:
yes, thanks